src/main/java/com/gojek/beast/converter/RowMapper.java

```diff
@@ -34,22 +34,21 @@ public RowMapper(ColumnMapping mappings) {
         if (mapping == null) {
             throw new ConfigurationException("BQ_PROTO_COLUMN_MAPPING is not configured");
         }
+
+        if (DynamicMessageUtil.isUnknownFieldExist(message)) {
+            statsClient.count("kafka.error.records.count,type=unknownfields," + statsClient.getBqTags(), 1);
```

Comment:
Log the message with a warn when when unknowfields exists

Reply:
will add log